XTS/sar/src/org/jboss/jbossts/xts/initialisation/CoordinatorSideInitialisation.java

```diff
@@ -25,6 +27,8 @@ public void startup() throws Exception
 
         ActivationCoordinatorInitialisation.startup();
         RegistrationCoordinatorInitialisation.startup();
+        RegistrationCoordinatorResponseInitialisation.startup();
```

Comment:
even `shutdown()` is empty method I expect it would be good to add the call of them to this class down here